userspace/libscap/scap.h

```diff
@@ -838,6 +838,11 @@ int32_t scap_get_stats(scap_t* handle, OUT scap_stats* stats);
 */
 int scap_get_modifies_state_ppm_sc(uint32_t * ppm_sc_array);
```

Comment:
Possibly, i'd specify expected length here too (and in `scap_get_modifies_state_tracepoints` too!)

Reply:
Done :)